database/sql/V066__auction_surplus_capturing_jit_order_owners.sql

```diff
@@ -0,0 +1,5 @@
+-- All surplus capturing JIT order owners that were part of an auction.
+CREATE TABLE surplus_capturing_jit_order_owners (
```

Comment:
I think this table is okay for now but I think eventually we should discuss collapsing a few tables into one. I believe we already have a lot of data in the informal `solver_competitions` table in a stricter format in other tables (e.g. `auction_prices`, `auction_participants`, `surplus_capturing_jit_order_owners`).

Reply:
I think this heavily depends on the way backend uses this data (but also accounting and other parties that have direct db access). If we stabilize the data and we see that these tables are always fetched together than I believe there is no a reason to keep them in separate tables.

Reply:
Totally agree. There is no action needed at the moment. Just wanted to point out that it might make sense to have a general discussion about the topic because we are accumulating data specifically for each individual auction in multiple different tables by now.
> If we stabilize the data and we see that these tables are always fetched together than I believe there is no a reason to keep them in separate tables.

Not sure I would make the "always fetched together" part a requirement for consolidating since these pieces of data at least conceptually fit together nicely and there is no harm in having  querying one half of the table data for one query and the other for another query. Overall these pieces of information are all things that we want to persist for every auction in case some component needs to know the state of the auction from the protocol's perspective.